searcher.py

```diff
@@ -32,8 +32,7 @@ def __call__(self, tokens):
 
 
 def search(df, userQuery):
-    # Defining constants for the data paths ***** MODIFY ACCORDINGLY *****
-    INDEX_DIR = "C:/UofT/4th_year/Capstone/Python_directory/schema"
+    INDEX_DIR = "index/"

```

Comment:
created a folder "index/" in that pull request, so we can point to it and none of us will have to update the path.